tests/src/ondevice_tests/img_transformation_test.cpp

```
@@ -43,6 +56,315 @@ inline std::array<std::array<float, 3>, 3> matmul(std::array<std::array<float, 3
+        projectedPoints.push_back({projectedPoint, p.depthMm, p.valid, p.charucoCornerId});
+    }
+
+    return projectedPoints;
+}
+
+std::vector<float> calculateProjectionErrors(const std::vector<CharucoPoint>& projectedPoints, const std::vector<CharucoPoint>& targetPoints) {
+    std::vector<float> errors;
+    for(const auto& projected : projectedPoints) {
+        auto it = std::find_if(targetPoints.begin(), targetPoints.end(), [&](const CharucoPoint& p) { return p.charucoCornerId == projected.charucoCornerId; });
+        if(it != targetPoints.end()) {
+            const float error = std::hypot(projected.coordinates.x - it->coordinates.x, projected.coordinates.y - it->coordinates.y);
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If DEPTHAI_NLOHMANN_DEFINE_TYPE_INTRUSIVE is a macro then please configure it.

(unknownMacro)